src/ChatWindow.vue

```diff
@@ -43,9 +44,21 @@ export default {
       type: Boolean,
       default: false
     },
+    /* Either agentProfile or agentProfiles is required */
```

Comment:
We need to determine which one of the two is required, we can't let this ambiguity :)

I'd say let's leave the array one required and when the chat is private, well, there will be only one participant 👍 

Do you agree?

Reply:
Yes!

My code tried to stay backward-compatible, but if we do not require backward-compatibility we can have something cleaner!